src/commands/referral/cleanupExpiredCode.ts

```diff
@@ -1,41 +1,27 @@
-import { ReferralCode } from '@prisma/client';
 import { getPrismaClient } from '../../clients';
-import { parseDate } from './parseDate';
+import { OpResult } from '../../utils/opResult';
 
-export const cleanupExpiredCode = (
-  referrals: ReferralCode[]
-): ReferralCode[] => {
+export const cleanupExpiredCode = async (): Promise<OpResult<unknown>> => {
   const prisma = getPrismaClient();
+  const currentDate = new Date();
+  try {
+    const result = await prisma.referralCode.deleteMany({
```

Comment:
Originally, this cleanup function takes in a bunch of referrals, filter out the expired ones, put on a task to clean them up and then return the good ones.

Instead of doing all of that, this now only deletes the expired ones.